src/Cache_Command.php

```diff
@@ -406,4 +409,168 @@ public function flush_group( $args, $assoc_args ) {
 		}
 		WP_CLI::success( "Cache group '$group' was flushed." );
 	}
+
+	/**
+	 * Get a nested value from the cache.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : Cache key.
+	 *
+	 * <key-path>...
+	 * : The name(s) of the keys within the value to locate the value to pluck.
+	 *
+	 * [--group=<group>]
+	 * : Method for grouping data within the cache which allows the same key to be used across groups.
+	 * ---
+	 * default: default
+	 * ---
+	 *
+	 * [--format=<format>]
+	 * : The output format of the value.
+	 * ---
+	 * default: plaintext
+	 * options:
+	 *   - plaintext
+	 *   - json
+	 *   - yaml
+	 * ---
+	 */
+	public function pluck( $args, $assoc_args ) {
+		list($key) = $args;
+		$group     = Utils\get_flag_value( $assoc_args, 'group' );
```

Comment:
```suggestion
		$group     = Utils\get_flag_value( $assoc_args, '' );
```

I think this should probably match `wp_cache_get()`'s default signature. Is there a reason you chose `'group'`?

Also, it'd be good to have functional tests covering the default group value vs. some custom group.

Reply:
I don't really understand the suggestion here.

The idea was to let the user specify a custom group for the cache when plucking or patching a key. Unfortunately, I couldn't use the same pattern as the other cache commands since the number of arguments can vary, so I've used an associative argument `--group` instead.

If I understand correctly, the `get_flag_value` function take the array of associative arguments as the first parameter and the name of the argument to retrieve in this array as the second parameter. That's why I used `group` here.

You can find a functional test for the custom group in [cache-pluck.feature:29](https://github.com/wp-cli/cache-command/pull/89/files#diff-578926fa26100c7b6a371df09de2a84fa25db326468404107b313a93997fae50R29).

Reply:
@petitphp You are correct. I misread the code and thought `'group'` was the default value being provided for `$group`.